### DISCUSSION TITLE
RNode SMA Connector

### DISCUSSION THREAD

**Discussion by hexjockey** [markqvist/Reticulum#194] [General]
I'm interested in purchased a LoRa module in order to build an RNode.
While searching for places to buy one, it would seem that it is much more
common to find units that have the SMA antenna connector on the PCB
rather than the u.FL (IPX).  I was looking at the build instructions and it
seems that a pigtail is used to convert the u.FL to an SMA antenna so,
short question long, is the reasoning for the build instructions to specify to use
a board with the u.FL antenna port just so it will fit into the existing case?  I can find a few
boards with the u.FL connector but they are priced around $80 CAD whereas
the boards with the SMA are generally around $30.

**Comment by markqvist** [comment]
Yes, the only reason is for the device to fit into the case. If you don't care about that, or want to make your own case (or even modify the available STL files), you can use the board version with built-in SMA connector **without any problems, and with no functional difference**.

So in short, there is no functional difference, it is *only* for it to fit into the case.

**Reply by hexjockey** [reply]
Thanks for the confirmation. In further digging, I now see that the units that
have the SMA connector also have the IPX connector right beside it. I did not
see that at first glance and thought the boards with came with IPX OR SMA but
not both.  I guess if I want to use the pigtail then and fit into the case that you
designed that I need to move a resistor on the board as per the guide although
it does not specify which resistor I need to move. Can you elaborate on which resistor
I would need to move if I did want to use the IPX connector?  I have being doing a lot of
reading on RNode and Reticulum and am quite excited about the possibility for use.
Thanks in advance!

**Reply by markqvist** [reply]
Yes, that is correct. They all have both connectors available. There's a 0 ohm resistor right next to the SMA and IPX connectors. You simply "rotate" the resistor 90 degrees to point at the IPX connector instead of the SMA connector. I'll grab a few photos and add them to the build guide shortly.

**Comment by markqvist** [comment]
I updated the build guide with some better explanations of the connectors, and photos to help identify the resistor to change: https://unsigned.io/guides/2023_01_14_Making_A_Handheld_RNode.html
